.golangci.yaml

```diff
@@ -0,0 +1,20 @@
+run:
+  concurrency: 4
+  deadline: 10m
+
+linters:
+  disable:
+  - unused
+  enable:
+  - revive
+
+issues:
+  exclude-use-default: false
+  exclude:
+  # revive
+  - var-naming # ((var|const|struct field|func) .* should be .*
+  - dot-imports # should not use dot imports
```

Comment:
Why would we forbid dot important. I guess there are handy for certain things e.g. unit testing

Reply:
I just copied the settings we have for our other repos. This part is on the exclude lists. You can see that the repo already is using dot-imports for gomega tests and still the check passes.